src/map/unit.hpp

```diff
@@ -27,6 +27,7 @@ struct unit_data {
 	struct skill_unit_group_tickset skillunittick[MAX_SKILLUNITGROUPTICKSET];
 	short attacktarget_lv;
 	short to_x, to_y;
+	unsigned char sx, sy; // Subtile position (0-15, with 8 being center of cell)
```

Comment:
You want to store a number and not a character.

Reply:
Char is a number from 0 to 255.

But I mainly use this because that's what clif.cpp expects to be put into the packet.

Reply:
Fixed that for you.